caterva2/services/sub.py

```diff
@@ -90,6 +89,7 @@ def __init__(self, abspath, path, metadata=None):
                     # This is a list, so we need to convert it to a string
                     dtype = eval(dtype)
                 self._dtype = np.dtype(dtype)
+                self._cparams = dict(metadata.schunk.cparams)
```

Comment:
FWIW, I recently added `cparams` as a property for `blosc2.NDArray`, so you could use `metadata.cparams` here too.

Reply:
Nope, `metadata` is another kind of object, so the comment above is invalid.